### PR TITLE
Fix Boston Globe support without cookie tampering, is relatively fast.

### PR DESCRIPTION
**Pull request by billatq**
Tested on Firefox Developer Edition. Didn't run into any issues, appears to work properly.

### CODE REVIEW

File: src/js/contentScript.js [hunk 1]
```diff
@@ -661,6 +661,25 @@ if (matchDomain('elmercurio.com')) {
...
+  if (paywall) {
+    removeDOMElement(paywall)
+  }
+  // Re-enable scrolling
+  const body = document.querySelector('body')
+  if (body) {
+    document.body.removeAttribute('style')
+  }
+  // Click the button to reveal the rest of the article
+  const button_div = document.querySelector('[id="continue_button"]')
```
Comment (iamadamdev) [inline src/js/contentScript.js:676]: nit: Use camelCasing.

File: src/js/contentScript.js [hunk 2]
```diff
@@ -661,6 +661,25 @@ if (matchDomain('elmercurio.com')) {
...
+  if (button_div) {
```
Comment (iamadamdev) [inline src/js/contentScript.js:677]: use camelCasing

**Review by iamadamdev — COMMENTED** [review]
Thanks for the PR. Can you also add the site to `manifest-ff.json`?

File: src/js/contentScript.js (hunk 1 above)
Comment (billatq) [reply src/js/contentScript.js:676]: > Thanks for the PR. Can you also add the site to `manifest-ff.json`?

I noticed that the site already is on `manifest-ff.json`, so I didn't touch that file.

Comment (billatq) [reply src/js/contentScript.js:676]: > nit: Use camelCasing.

Will update.

File: src/js/contentScript.js (hunk 2 above)
Comment (billatq) [reply src/js/contentScript.js:677]: > use camelCasing

Will update.

**Comment by billatq** [comment]
Updated with changes responsive to feedback.

Note that `manifest-ff.json` already has the `bostonglobe.com`: https://github.com/iamadamdev/bypass-paywalls-chrome/blob/167d1420d37076c5380e87381fae69e4c703685a/manifest-ff.json#L131

**Comment by iamadamdev** [comment]
It's in permissions but not `content_scripts` sections of manifest-ff.json

**Comment by billatq** [comment]
> It's in permissions but not `content_scripts` sections of manifest-ff.json

Fixed

**Review by iamadamdev — APPROVED** [review]

**Pull request merged**